packages/table/preview/index.tsx

```diff
@@ -150,7 +150,7 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
     public constructor(props: any, context?: any) {
         super(props, context);
 
-        this.state = {
+        const stateDefaults = {
```

Comment:
well this no longer belongs in the constructor.

define `const DEFAULT_STATE` outside class.